src/main/java/com/blackducksoftware/integration/hub/rest/CredentialsRestConnection.java

```diff
@@ -105,7 +105,7 @@ public void clientAuthenticate()
                         response = getClient().newCall(request).execute();
                         logResponseHeaders(response);
                         if (!response.isSuccessful()) {
-                            throw new HubIntegrationException(response.message());
+                            throw new HubIntegrationException(String.format("Connection Error: %1$s %2$s", response.code(), response.message()));
```

Comment:
The %1$s should be changed to %s to be consistent everywhere, unless you are specifically using the same segment multiple times or out of order